plugins/config/config_test.go

```diff
@@ -0,0 +1,150 @@
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package config_test
+
+import (
+	"testing"
+
+	"github.com/spf13/cobra"
+
+	"github.com/ligato/networkservicemesh/plugins/config"
+	"github.com/ligato/networkservicemesh/utils/command"
+	. "github.com/onsi/gomega"
+)
+
+type Config1 struct {
+	One string
+	Two string
+}
+
+func TestLoadConfigDefault(t *testing.T) {
+	RegisterTestingT(t)
+	cmd := &cobra.Command{Use: "command"}
+	command.SetRootCmd(cmd)
+	plugin := config.NewPlugin()
+	Expect(plugin).NotTo(BeNil())
+	Expect(plugin.Deps.Name).To(Equal(config.DefaultName))
+	Expect(plugin.Deps.DefaultConfig).ToNot(BeNil())
+	Expect(plugin.Deps.Cmd).To(Equal(cmd))
+	err := plugin.Init()
+	Expect(err).To(BeNil())
+	config := plugin.LoadConfig()
```

Comment:
We're importing a config package and creating a config variable. We should either rename this variable or alias the config package under a new name.